packages/i18n/src/locales/en/options/shelly.yml

```
@@ -0,0 +1,64 @@
+---
+straightSides:
+  title: Straight sides
+  description: Forms a tubular shirt, which is a shirt where the hips and chests are the same width and connected by a straight seam.
+
+bodyHem:
+  title: Body hem
+  description: Sets the hem allowance along the bottom of the body of the shirt, as a multiple of the seam allowance.
+
+bodyLength:
+  title: Body length
+  description: Controls how long the shirt is.
+
+sideShape:
+  title: Body shape
+  description: Controls how the side seam is curved. This option is ignored if the straight sides option is enabled.
+
+chestEase:
+  title: Chest ease
+  description: Controls how wide the shirt will be at your chest.
+
+hipsEase:
+  title: Hips ease
+  description: Controls how wide your shirt will be at the bottom hem. This option is only used if the straight sides option is disabled.
+
+neckbandLength:
+  title: Neckband length
+  description: Controls how long the neckband is, as a percentage of the circumferance of the neckline.
+
+neckbandWidth:
+  title: Neckband width
+  description: Controls how wide the finished neckband will be, as a percentage of the neckline circumference.
+
+neckEase:
+  title: Neck ease
+  description: Controls how large the neck hole is.
+
+neckBalance:
+  title: Neck position
+  description: Controls where the neck hole is placed on the shirt.
+
+raglanScoopLength:
+  title: Raglan scoop length
+  description: Controls how far out from the armhole the raglan seam starts to scoop.
+
+raglanScoopMagnitude:
+  title: Raglan scoop depth
+  description: Controls how much extra material to place under the armpits.
+
+sleeveEase:
+  title: Sleeve ease
+  description: Affects how wide the sleeve is at the armhole on all shirts, and has some affect on the width at the hem of short sleeved shirts.
+
+sleeveHem:
+  title: Sleeve hem
+  description: Sets the hem allowance of the hem along the end of the sleeves, as a multiple of the seam allowance.
+
+sleeveLength:
+  title: Sleeve length
+  description: Controls the how long the sleeves of your shirt will be.
+
+wristEase:
+  title: Wrist ease
+  description: Affects how wide the end of the sleeve is on sleeves that extend past the biceps. The effect is greater on longer sleeves.
```

```suggestion
  description: Affects how wide the end of the sleeve is on sleeves that extend past the biceps. The effect is greater on longer sleeves.
```

(I attempted to add a newline character to the end of the line.)